packages/lib-classifier/src/plugins/tasks/survey/components/SurveyTask.js

```diff
@@ -72,4 +130,4 @@ SurveyTask.propTypes = {
   }).isRequired
 }
```

Comment:
Maybe document the task props with docgen comments?

Reply:
I attempted to, could not figure out why they weren't showing, and it turned out it was because I was importing the survey task via plugins and to generate docgen comments I needed to import the survey task component colocated with story 🤦‍♂️ , so this change fixed it 👍 :
```
import SurveyTask from '@plugins/tasks/survey'

import SurveyTask from './SurveyTask'
```